_posts/2022-10-18-release.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Release 2022.42
+author: Priya
+date: 2022-10-18
+category: Jekyll
+layout: post
+---
+
+### Defect fixes
+
+* Fixed an issue where the dataflow graph and apply template took a long time to appear on the preview panel.
+* Resolved a bug where the task description was displayed in an empty format.
```

Comment:
In general call things defect instead of bug

Reply:
okay, I fixed the same in 2022.42 and 2022.42.2